openfisca_aotearoa/variables/acts/super/super.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and entitlement system
+from openfisca_aotearoa.entities import Person
+
+
+class super__living_alone(Variable):
+    value_type = int
+    entity = Person
+    definition_period = MONTH
+    label = u"Is a person considered as living alone"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0084/latest/whole.html#DLM5578822"
+
+
+class super__is_receiving_compensation(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Is receiving compensation payment through ACC"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0049/latest/DLM105404.html#DLM105404"
+
+
+class super__has_partner(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Has a partner"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0084/latest/DLM114223.html"
+
+
+class super__is_veteran(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Applicant is a veteran"
+    reference = "http://www.legislation.govt.nz/act/public/2001/0084/latest/DLM115004.html"
+
+
+class super__eligibility(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Classified as eligible for NZ Super"
+
+    def formula(persons, period, parameters):
+        return (persons('age', period) >= 65) *\
+            not_(persons('super__is_receiving_compensation', period)) +\
```

Comment:
Just checking, I would have thought super__has_partner would be referenced in here? Or is that variable only needed for the calculation of the amount?

Reply:
@verbman yes you're correct, its only needed for calculation.